dds/DCPS/NetworkConfigUpdater.h

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
+#define OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
+
+#include "ace/config.h"
+
+#ifdef ACE_HAS_IOS 
```

Comment:
If this will compile without iOS, then I would remove the ifdef and always compile it and the cpp.